src/main/scala/com/fortysevendeg/translatebubble/modules/clipboard/impl/ClipboardServicesComponentImpl.scala

```diff
@@ -103,6 +103,18 @@ trait ClipboardServicesComponentImpl
 
   }
 
+  private def isValidText(text: CharSequence): Boolean = {
+    !isTrimmedTextEmpty(text) && !isTextANumber(text) && !isTextAUrl(text)
+  }
+
+  private def isTrimmedTextEmpty(text: CharSequence): Boolean = text.toString.trim.length == 0
```

Comment:
This is the previous implementation. I've just moved the code to a private function. If you suggest me any better approach to implement this function I can change it :)